db/schema.rb

```diff
@@ -10,9 +10,8 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2024_01_17_115443) do
+ActiveRecord::Schema[7.0].define(version: 2024_05_29_212213) do
   # These are extensions that must be enabled in order to support this database
-  enable_extension "pgcrypto"
```

Comment:
This was automatically removed after running db migrate. It was added when we introduced automatic uuids for sharable lesson previews, and that still works on the review app so I'm happy for it to be binned 🗑️